src/hooks/index.ts

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from 'react';
+import {Configuration, Project, V0alpha0Api} from "@ory/client";
+
+const sdk = new V0alpha0Api(new Configuration({
+  basePath: 'https://api.console.ory:8080',
+  baseOptions: {
+    withCredentials: false,
+  }
+}))
+
+export function getSdkUrl() {
+  const [project, setProject] = useState<Project | undefined>();
+
+  useEffect(() => {
+    sdk.listProjects().then(({data: projects}) => {
+      if (projects.length > 0) {
+        setProject(projects[0]);
```

Comment:
What about `project[n]`? Still WIP?

It just reminds me of a similar thing (unrelated to Ory :) ) it took me a while to fix.